packages/schematics/schematics/blank/factory.ts

```diff
@@ -102,7 +102,7 @@ export default function (options: Schema): Rule {
     try {
       const packageJsonContent = tree.read('/package.json');
       if (packageJsonContent) {
-        const packageJson = JSON.parse(packageJsonContent.toString('utf-8'));
+        const packageJson = JSON.parse(packageJsonContent.toString('utf-8')) as {};
```

Comment:
This is for the linter? I'm surprised it runs at all on third party code.

We should add a comment here to explain that we need to cast the result for the linter. Also consider using `unknown` as a safer alternative to `{}` if not too much trouble.

Reply:
Sorry, to be exact, it is the compiler, not the linter. go/tsetse is built into the compiler, so the errors show up as hard errors at compile time.
I added the comments and typed it as `never` instead of `{}`. I'm surprised that bracket notation is allowed on `never` type.